### PR TITLE
Solves issue - #126

### PR DESCRIPTION
**Pull request by SANJAY072000**
Kindly review this PR to issue #126. Resolves smooth scroll error on safari.

### CODE REVIEW

File: css/custom.css [hunk 1]
```diff
@@ -45,7 +45,7 @@
 }
 
 html{
-	scroll-behavior: smooth;
+	/* scroll-behavior: smooth; */
```
Comment (sleepypioneer) [inline css/custom.css:48]: If this is no longer need, I would suggest to remove it as leaving it here could become confusing, if you think it really should stay could you add a comment to why that is?

File: index.html [hunk 2]
```diff
@@ -474,6 +516,12 @@ <h3>Programming Diversity</h3>
                 };
 				 return t;
             }(document, "script", "twitter-wjs"));
+
```
Comment (sleepypioneer) [inline index.html:519]: is there a reasoning behind this amount of blank spaces? Perhaps left in while developing?

File: index.html [hunk 3]
```diff
@@ -456,6 +456,48 @@ <h3>Programming Diversity</h3>
     			$('.first-button').on('click', function () {
   				$('.animated-icon').toggleClass('open');
     			});
+
+// added code to resolve safari smooth scrolling error
```
Comment (sleepypioneer) [inline index.html:460]: perhaps an approach here could be to add the issue number this code resolves? I am not familiar with the practice of adding a comment to note that code was added to resolve something specific but it seems useful :)

**Review by sleepypioneer — COMMENTED** [review]
Thanks for you contribution @SANJAY072000 I left a few comments. I am not overly familiar with the code it's self but it looks good to me, I wondered if you might make a small screen capture to show what this fixes (ie a before and after video) which would make it easier to give this a thumbs up :)

**Comment by jonatoni** [comment]
Closing it since it's an old PR - but if you are still interested in working on this @SANJAY072000 and replying to @sleepypioneer comments, please feel free to open it again @SANJAY072000 :smiley:

**Pull request closed without merging**